core/indigo-core/molecule/base_molecule.h

```diff
@@ -466,6 +484,7 @@ namespace indigo
         // When molecule gets edited then edit revision is increased.
         // If edit revision is the same then molecule wasn't edited
         int _edit_revision;
+        PtrArray<MetaObject> _meta_data;
```

Comment:
We have a chance to get rid of hand-made PtrArray ;)

Reply:
We can't do it in BaseMolecule class.